src/pages/Login/Login.js

```diff
@@ -1,8 +1,82 @@
-import React from 'react'
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import React, { useState } from 'react'
+import './Login.scss'
 
 const Login = () => {
-  return <h1>hello</h1>
+  const [email, setEmail] = useState('')
+  const [pw, setPw] = useState('')
+  const [emailError, setEmailError] = useState(false)
+  const [pwError, setPwError] = useState(false)
+
+  const signIn = () => {
+    fetch('', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json;charset=utf-8' },
+      body: JSON.stringify({
+        email: email,
+        password: pw,
+      }),
+    })
+      .then(response => {
+        return response.json()
+      })
+      .then(response => {
+        if (response.data === null) {
+          alert('로그인실패')
+        } else {
+          alert('로그인성공')
+          localStorage.setItem('access_token', response.data.accessToken)
+        }
+      })
+  }
+
+  return (
+    <div className="signIn">
+      <h1 className="loginTitle">로그인</h1>
+      <ul className="loginInputContainer">
+        <li className="inputBox">
+          <input
+            autoFocus={true}
+            className={emailError ? 'inputError' : 'input'}
+            type="text"
+            placeholder="이메일"
+            onChange={e => {
+              setEmail(e.target.value)
+              {
+                e.target.value === ''
+                  ? setEmailError(true)
+                  : setEmailError(false)
+              }
+            }}
+          ></input>
+        </li>
+        {emailError ? <p className="errorMsg">이메일을 입력해주세요</p> : null}
```

Comment:
```suggestion
        {emailError && <p className="errorMsg">이메일을 입력해주세요</p>}
```

true일때만 UI가 보여지는 형태일때는 && 연산자를 활용해주세요